src/twisted/internet/test/test_resolver.py

```diff
@@ -538,7 +541,8 @@ def test_portNumber(self):
         self.assertEqual(receiver._ended, True)
 
 
-class JustEnoughReactor(ReactorBase, object):
+
+class JustEnoughReactor(ReactorBase):
     """
```

Comment:
Not knowing why there is an `object` in the "old" inheritence explicitly, is this removed because `ReactorBase` now inherits from the new class, which inherits from `object` -- and thus there's at least one `object` in the tree, so python2 should be happy...?

Reply:
ReactorBase already inherited from object when JustEnoughReactor was first added so I'm not sure.  I think it was probably a mistake, but a harmless one (because having object in your hierarchy multiple times is the normal case).  I probably didn't need to remove it here but I was trying to remove `JustEnoughReactor` entirely and when that didn't work I put it back, but apparently not all of it. :)